[PFCMP-0011] Empty View

SetyaG opened the pull request:
Closes #111 
***
## Description
Create the empty view component to be used when there's no data found

## Related Links
List related PRs against other branches:

List | link
------ | ------
User Story | [link]()
Figma | [link]()
Issue | [link](#111)

## Todos
- [X] UI
- [ ] Coordinator
- [ ] Network
- [ ] Tests
- [ ] Documentation

## Screenshot / Video
Screenshot consisting of changes made on this branch.

### UI Changes
UI related changes will be shown here (can be either screenshot or video format)

### Network Status
Network related changes will be shown here (screenshot of request status)

### Test Coverage
Test coverage result for designated changes will be shown here (screenshot of coverage)

## Notes
Debt: 
- Elements somehow still don't exactly positioned at the center

SetyaG committed:
Created Empty View
Added function for changing text and image of the empty view
Created Empty View Constant

inxudianz commented:
Please provide picture by testing it a random viewController

SetyaG commented:
<img width="317" alt="Screen Shot 2020-11-18 at 19 23 51" src="https://user-images.githubusercontent.com/45433184/99524690-c4b58200-29d3-11eb-820a-49f56761d9d0.png">

SetyaG commented:
The sdasda sentence is the text, the image should be above but somehow it doesn't appear

inxudianz commented on Component/Component/View/EmptyView/FamiliEmptyView.swift:
```diff
@@ -0,0 +1,106 @@
+    
+    @IBInspectable var imageName: String = "" {
+        didSet {
+            guard let image = UIImage(named: imageName) else { return }
```
use UIImage(named: , in: ,compatibleWith:) initialization to make image visible

SetyaG replied:
updated

SetyaG committed:
Updated the inspectable var into UIImage type
Updated the setImage function to be able to capture the Bundle for the specified image group

Addition: Updated some of the documentation for the view functions

SetyaG commented:
<img width="317" alt="Screen Shot 2020-11-20 at 14 19 42" src="https://user-images.githubusercontent.com/45433184/99766440-10833b00-2b3c-11eb-9597-226e50ff29e4.png">

Image now can be shown

inxudianz committed:
Merge branch 'develop' into component/PFCMP-0011
